src/site/content/en/react/how-to-use-amp-in-nextjs-and-why/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: post
+title: How to use AMP in Next.js (and why)
+authors:
+  - houssein
+subhead: |
+  Add AMP to any of your Next.js pages to guarantee fast page loads
+date: 2019-10-17
+draft: true
+---
+
+[AMP](https://amp.dev) (Accelerated Mobile Pages) is a web component framework that guarantees
+that your pages will load fast. [Next.js][intro] has built-in support for AMP.
```

Comment:
s/"that guarantees that your pages will load fast"/"that guarantees fast page loads"?